src/log-pcap.c

```diff
@@ -1698,6 +1912,15 @@ static int PcapLogOpenFileCtx(PcapLogData *pl)
     return -1;
 }
 
+char *PcapLogGetFilename(void)
+{
+    /* do we have a file per thread */
+    if (pcap_file_multi != NULL) {
+        return pcap_file_multi;
+    }
+    return NULL;
```

Comment:
Can we get this in non-multi mode as well?  With rotation, its still valuable to have no matter what, and consistent.